app/plugins/adp-backend/package.json

```diff
@@ -23,22 +23,30 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.20.0",
+    "@backstage/backend-common": "^0.19.8",
+    "@backstage/backend-plugin-api": "^0.6.7",
     "@backstage/config": "^1.1.1",
+    "@backstage/errors": "^1.2.3",
+    "@backstage/plugin-auth-node": "^0.4.1",
     "@types/express": "*",
+    "@types/uuid": "^9.0.7",
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
-    "winston": "^3.2.1",
```

Comment:
what library are you using for logging messages, since this one has been removed?